lib/main.dart

```diff
@@ -1,125 +1,141 @@
-          // horizontal).
-          //
-          // TRY THIS: Invoke "debug painting" (choose the "Toggle Debug Paint"
-          // action in the IDE, or press "p" in the console), to see the
-          // wireframe for each widget.
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            const Text(
-              'You have pushed the button this many times:',
-            ),
-            Text(
-              '$_counter',
-              style: Theme.of(context).textTheme.headlineMedium,
+      home: Scaffold(
+        backgroundColor: Colors.cyan,
+        appBar: AppBar(
+          title: Text(
+            movie.title,
+          ),
+          centerTitle: true,
+        ),
+        body: SingleChildScrollView(
+          child: SafeArea(
+            child: Center(
+              child: Column(
+                children: [
+                  Stack(
+                    children: [
+                      Image.network(movie.backdrop),
+                      Positioned(
+                        bottom: UiConstants.firstPositionedBottom,
+                        right: UiConstants.firstPositionedRight,
+                        child: TitleWidget(
+                          movie: movie,
+                        ),
+                      ),
+                    ],
+                  ),
+                  Padding(
+                    padding: const EdgeInsets.all(UiConstants.padding16),
+                    child: Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                      children: [
+                        MiddleText(
+                            text:
+                                '${UiConstants.textConstant} ${movie.releaseDate}'),
+                        MiddleText(text: movie.voteAverage),
+                      ],
+                    ),
+                  ),
+                  Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                      children: movie.genres
+                          .map(
+                            (genre) => Container(
+                              decoration: const BoxDecoration(
+                                color: Colors.black12,
+                                borderRadius: BorderRadius.all(
+                                  Radius.circular(UiConstants.borderRadius),
+                                ),
+                              ),
+                              child: Padding(
+                                padding:
+                                    const EdgeInsets.all(UiConstants.padding8),
+                                child: Text(
+                                  genre,
+                                  style: CustomTextStyle.customTextStyleWhite,
+                                ),
+                              ),
+                            ),
+                          )
+                          .toList()),
```

Comment:
You could extract this as a separate widget, or use the Material Design `Chip` widget!